c++/src/kj/async-test.c++

```diff
@@ -25,6 +25,19 @@
 #include "mutex.h"
 #include "thread.h"
 
+// TODO(later): Refactor this more cleanly so that the KJ library itself defines this.
+#if !__BIONIC__ && !__OpenBSD__
```

Comment:
We haven't actually fixed `async.c++` yet to recognize that OpenBSD doesn't have fibers, so this is inconsistent with the check being performed there...